src/style.css

```diff
@@ -0,0 +1,115 @@
+* {
+  margin: 0;
+}

```

Comment:
```suggestion
* {
  margin: 0;
}
```

Forbidden to set margin in `*` selector